gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/GherkinTestRunner.cs

```diff
@@ -182,16 +182,12 @@ private void OutputResults(List<ResultFeature> results)
                 foreach (var resultScenario in resultFeature.Scenarios)
                 {
                     totalScenarios++;
-                    WriteOutput($"  Scenario: {resultScenario.Key.Name}");
```

Comment:
This shortens the output to make it easier to see which scenarios are actually failing, right?

Reply:
yeah - added a note about that to the commit comment. it was getting way too hard to find the errors.

Reply:
Yeah makes sense, I've also ran into that problem in the past.